cluster/pre-existing/util.sh

```diff
@@ -44,19 +44,10 @@ function create-certs {
   rm /tmp/kubeconfig
 
   execute-cmd-on-pre-existing-master-with-retries "sudo cat /etc/kubernetes/admin.conf" > /tmp/kubeconfig
-  CA_CERT_BASE64=$(cat /tmp/kubeconfig | grep certificate-authority | awk '{print $2}' | head -n 1)
-  KUBELET_CERT_BASE64=$(cat /tmp/kubeconfig | grep client-certificate-data | awk '{print $2}' | head -n 1)
-  KUBELET_KEY_BASE64=$(cat /tmp/kubeconfig | grep client-key-data | awk '{print $2}' | head -n 1)
-
-  # Local kubeconfig.kubemark vars
-  KUBECFG_CERT_BASE64="${KUBELET_CERT_BASE64}"
-  KUBECFG_KEY_BASE64="${KUBELET_KEY_BASE64}"
-
-  # The pre-existing Kubernetes master already has these setup
-  # Set these vars but don't use them
-  CA_KEY_BASE64=$(dd if=/dev/urandom bs=128 count=1 2>/dev/null | base64 | tr -d "=+/" | dd bs=32 count=1 2>/dev/null)
-  MASTER_CERT_BASE64=$(dd if=/dev/urandom bs=128 count=1 2>/dev/null | base64 | tr -d "=+/" | dd bs=32 count=1 2>/dev/null)
-  MASTER_KEY_BASE64=$(dd if=/dev/urandom bs=128 count=1 2>/dev/null | base64 | tr -d "=+/" | dd bs=32 count=1 2>/dev/null)
-  KUBEAPISERVER_CERT_BASE64=$(dd if=/dev/urandom bs=128 count=1 2>/dev/null | base64 | tr -d "=+/" | dd bs=32 count=1 2>/dev/null)
-  KUBEAPISERVER_KEY_BASE64=$(dd if=/dev/urandom bs=128 count=1 2>/dev/null | base64 | tr -d "=+/" | dd bs=32 count=1 2>/dev/null)
+  CA_CERT_BASE64=$(grep certificate-authority /tmp/kubeconfig | awk '{print $2}' | head -n 1)
```

Comment:
can we add a comment about what uses these?

Reply:
I think `CA_CERT_BASE64` defined at this line is not used by anywhere.

Because, `cluster/gce/util.sh` use it, but it is defined by `cluster/gce/util.sh`
https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/util.sh#L1598
Also, `test/kubemark/iks/startup.sh` and `test/kubemark/iks/util.s` use it, but `create-certs` which defines `CA_CERT_BASE64` is not used by those files.

From these things, I will remove `create_certs` in this PR. Do we need to add any comments to the file?

Reply:
if we remove it we don't need comments, but if we're exporting something ideally we note that so we can eliminate it in the future potentially (or avoid doing so because we already did the work to discover what depends on it!)

Reply:
I updated to leave those variables safely and add comments to the file.
Could you check it?